### PR TITLE
chore: rename "tab-group" to "tabs"

### PR DESCRIPTION
**Pull request by jelbourn**
Also adds a `package.json` for tabs

### CODE REVIEW

File: typings.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
   },
   "globalDependencies": {
     "core-js": "registry:dt/core-js#0.0.0+20160317120654",
-    "es6-shim": "registry:dt/es6-shim#0.31.2+20160317120654",
```
Comment (robertmesserle) [inline typings.json]: Unrelated change?

**Comment by robertmesserle** [comment]
LGTM - just added one comment making sure the typings change was intentional for this PR.

File: typings.json (hunk 1 above)
Comment (jelbourn) [inline typings.json]: Just a minor "oh, that needs to go" change.
